integration_tests/__tests__/clear_cache.test.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
```

Comment:
change the license header to be MIT, please

Reply:
Also found jest/integration_tests/__tests__/pass_with_no_tests.test.js with BSD license. Should I take care of that here?

Reply:
Yes please